### DISCUSSION TITLE
Scheduled weights

### DISCUSSION THREAD

**Discussion by cubiq** [cubiq/ComfyUI_IPAdapter_plus#471] [Announcements]
It's now possible to schedule weights, you can do that by converting the weight widget to an input like so

![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/e69282d3-bbfd-4371-8b74-96889c1b4a37)

The `IPAdapter Weights` helps you generating simple transition. It will change in the future but for now it works.

Before you had to use faded masks, now you can use weights directly which is lighter and more efficient. The animation below has been done with just IPAdapter and no controlnet or masks.

https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/f19a557b-33d4-48ff-bf81-aa543ea8d344

This is the workflow: [weights.json](https://github.com/cubiq/ComfyUI_IPAdapter_plus/files/14967446/weights.json)

**Comment by PetitPrinceb612** [comment]
Thank you very much, the effect is great. I have just learned SD and have a question. What is the main function of embeds_scaling in iadapter batch?

**Comment by aidiffuser** [comment]
Hi Don Matteo, I hope all is well.

Is there a way to implement weight keyframing? Is it possible? In the sense that I could set a specific weight interpolation for, say, a number of frames at the beggining of an animation and a different weight interpolation for the last part? 

Kinda like how the prompt travel works.

Thank you!
